src/components/scenegraph/SceneGraph.js

```diff
@@ -70,19 +66,16 @@ export default class SceneGraph extends React.Component {
     for (let i = 0; i < this.state.filteredEntities.length; i++) {
       const entityOption = this.state.filteredEntities[i];
       if (entityOption.entity === entity) {
-        this.setState({ selectedEntity: entity, selectedIndex: i });
+        this.setState({ selectedIndex: i });
         // Make sure selected value is visible in scenegraph
         this.expandToRoot(entity);
-        if (this.props.onChange) {
-          this.props.onChange(entity);
-        }
-        Events.emit('entityselect', entity, true);
```

Comment:
This true param here has no meaning, there is no second parameter in any of the entityselect listener in the code base.